packages/shared/src/lib/reminderFields.ts

```diff
@@ -5,6 +5,16 @@ export interface ReminderFields {
     reminderOption?: string;
 }
 
+const thanksgivingUsa = (year: number = new Date().getFullYear()): Date => {
+    const novemberFirst = new Date(year, 10, 1).getDay();
+
+    return new Date(year, 10, 22 + ((11 - novemberFirst) % 7));
```

Comment:
Would it be worth adding a comment about _why_ this logic works? I'm struggling to get my head around it!

(As an aside: I don't think I'll ever not get confused for a second before remembering that the month value here is 0 based 😂 )

Reply:
Yeah, the 0-based month numbers are a nightmare. I'll add some comments to explain this function.